nixos/common/nix.nix

```diff
@@ -13,6 +13,9 @@
   # The default at 10 is rarely enough.
   nix.settings.log-lines = lib.mkDefault 25;
 
+  # De-duplicate store paths using hardlinks
+  nix.settings.auto-optimise-store = lib.mkDefault true;
```

Comment:
In my experience this still slows down copies of store paths significantly. IMO the systemd timer is a better alternative which doesn't slow down every store operation.

Reply:
Makes sense. We can have the timer also on both platforms.

Reply:
Is there an option for this?

Reply:
yep https://search.nixos.org/options?channel=unstable&show=nix.optimise.automatic&from=0&size=50&sort=relevance&type=packages&query=nix.optimise

Reply:
Applied.